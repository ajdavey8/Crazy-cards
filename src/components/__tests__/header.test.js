import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Header from '../header';

describe('Header', () => {
  const wrapper = shallow(<Header />);

  it('renders and matches snapshot', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders the header', () => {
    expect(wrapper.find('.header h1').text()).toContain('Crazy Cards');
    expect(wrapper.find('.header p').text()).toContain(
      'Fill in your details to see which cards you are eligble for'
    );
  });
});
