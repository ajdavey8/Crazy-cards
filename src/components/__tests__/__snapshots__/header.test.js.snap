// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders and matches snapshot 1`] = `
<div
  className="header"
>
  <h1>
    Crazy Cards
  </h1>
  <p>
    Fill in your details to see which cards you are eligble for
  </p>
</div>
`;
